spec/support/features.rb

```diff
@@ -1,6 +1,5 @@
 # spec/support/features.rb
 require File.expand_path('../features/session_helpers', __FILE__)
-require File.expand_path('../fixture_helpers', __FILE__)
```

Comment:
I removed them just because they were no longer being used anywhere, but I'll add them back in if you think they are still useful.